core/src/test/java/com/linecorp/armeria/server/VirtualHostBuilderTest.java

```diff
@@ -327,4 +328,16 @@ void precedenceOfDuplicateRoute() {
         final Route route = serviceConfig.route();
         assertThat(route).isSameAs(routeA);
     }
+
+    @Test
+    void multipartUploadsLocationCustomization() {
+        final VirtualHost h1 = new VirtualHostBuilder(Server.builder(), false)
+                .multipartUploadsLocation(FileSystems.getDefault().getPath("logs", "access.log"))
+                .build(template, noopDependencyInjector);
+        assertThat(h1.multipartUploadsLocation().getFileName().toString()).isEqualTo("access.log");
+
+        final VirtualHost h2 = new VirtualHostBuilder(Server.builder(), false)
+                .build(template, noopDependencyInjector);
+        assertThat(h2.multipartUploadsLocation().getFileName().toString()).isEqualTo("multipart-uploads");
```

Comment:
Because the default location may change, you could:
```suggestion
        assertThat(h2.multipartUploadsLocation()).isEqualTo(template.multipartUploadsLocation);```

If `multipartUploadsLocation` is private, you'll have to relax the visibility to package-private and annotate it with `@VisibleForTesting`.

Reply:
Done 👍